msgid ""
msgstr ""
"Project-Id-Version: openplotter\n"
"POT-Creation-Date: 2021-01-17 13:13+0100\n"
"PO-Revision-Date: 2021-11-25 18:05\n"
"Last-Translator: \n"
"Language-Team: Catalan\n"
"Language: ca\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 1.8.7.1\n"
"X-Poedit-Basepath: ../../..\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Poedit-SearchPath-0: .\n"
"X-Crowdin-Project: openplotter\n"
"X-Crowdin-Project-ID: 289888\n"
"X-Crowdin-Language: ca\n"
"X-Crowdin-File: openplotter-network.po\n"
"X-Crowdin-File-ID: 14\n"

#: networkPostInstall.py:30
msgid "Installing python packages..."
msgstr "Instal·lant paquets python..."

#: networkPostInstall.py:33 networkPostInstall.py:51 networkPostInstall.py:63
#: networkPostInstall.py:69 networkPreUninstall.py:31 networkPreUninstall.py:62
msgid "DONE"
msgstr "FET"

#: networkPostInstall.py:34 networkPostInstall.py:52 networkPostInstall.py:64
#: networkPostInstall.py:70 networkPreUninstall.py:32 networkPreUninstall.py:57
#: networkPreUninstall.py:63
msgid "FAILED: "
msgstr "ERROR: "

#: networkPostInstall.py:37
msgid "Editing config files..."
msgstr "Editant fitxers de configuració..."

#: networkPostInstall.py:55
msgid "Enabling services..."
msgstr "Habilitant serveis..."

#: networkPostInstall.py:66
msgid "Setting version..."
msgstr "Establint la versió..."

#: networkPreUninstall.py:28
msgid "Removing python packages..."
msgstr "Eliminant paquets python..."

#: networkPreUninstall.py:35
msgid "Restoring network Debian defaults..."
msgstr "Reestablint els valors de xarxa per defecte de Debian..."

#: networkPreUninstall.py:48
msgid "Removing openplotter-network service..."
msgstr "Eliminant servei openplotter-network-service..."

#: networkPreUninstall.py:56
msgid "DONE. PLEASE REBOOT TO RESTORE NETWORK SETTINGS"
msgstr "FET. SI US PLAU REINICIA PER RESTAURAR LES PREFERÈNCIES DE XARXA"

#: networkPreUninstall.py:59
msgid "Removing version..."
msgstr "Eliminant versió..."

#: openplotterNetwork.py:38
msgid "Network"
msgstr "Xarxa"

#: openplotterNetwork.py:48
msgid "Help"
msgstr "Ajuda"

#: openplotterNetwork.py:51
msgid "Settings"
msgstr "Preferències"

#: openplotterNetwork.py:54
msgid "Addresses and Ports"
msgstr "Adreces i ports"

#: openplotterNetwork.py:56
msgid "Check Network"
msgstr "Comprova la xarxa"

#: openplotterNetwork.py:59
msgid "Install Wifi Drivers"
msgstr "Instal·lant els controladors Wifi"

#: openplotterNetwork.py:59
msgid "This does only help for unrecognized usb wlan type:"
msgstr "Això només és útil per a tipus de wlan usb no reconeguts:"

#: openplotterNetwork.py:66
msgid "Access Point"
msgstr "Punt d'accés"

#: openplotterNetwork.py:134
msgid "Network Mode"
msgstr "Mode de xarxa"

#: openplotterNetwork.py:139
msgid "AP"
msgstr "Punt d'accés"

#: openplotterNetwork.py:149
msgid "5 GHz"
msgstr "5 GHz"

#: openplotterNetwork.py:151
msgid "Add Ethernet port to the AP"
msgstr "Afegir el port Ethernet a l'AP"

#: openplotterNetwork.py:174
msgid "Access Point Settings"
msgstr "Configuració del punt d'accés"

#: openplotterNetwork.py:177
msgid "Sharing Internet device"
msgstr "Compartint dispositiu d'internet"

#: openplotterNetwork.py:185
msgid "SSID \n"
"maximum 32 characters"
msgstr "SSID \n"
"màxim 32 caràcters"

#: openplotterNetwork.py:193
msgid "Password \n"
"minimum 8 characters required"
msgstr "Contrasenya \n"
"mínim 8 caràcters"

#: openplotterNetwork.py:202
msgid "Channel"
msgstr "Canal"

#: openplotterNetwork.py:312 openplotterNetwork.py:332
#: openplotterNetwork.py:337 openplotterNetwork.py:387
#: openplotterNetwork.py:391 openplotterNetwork.py:433
#: openplotterNetwork.py:466 openplotterNetwork.py:484
#: openplotterNetwork.py:614
msgid "none"
msgstr "cap"

#: openplotterNetwork.py:404
msgid "Wait please... "
msgstr "Si us plau espera... "

#: openplotterNetwork.py:538
msgid "Your SSID must have a maximum of 32 characters and your password a minimum of 8."
msgstr "El teu SSID ha de tenir un màxim de 32 caràcters i la contrasenya un mínim de 8."

#: openplotterNetwork.py:589
msgid "OpenPlotter will reboot. If something goes wrong and you are on a headless system, you may not be able to reconnect again.\n\n"
"Are you sure?"
msgstr "OpenPlotter es reiniciarà. Si alguna cosa no va bé i estàs en un sistema sense monitor, és possible que no et puguis tornar a connectar.\n\n"
"Estàs segur?"

#: openplotterNetwork.py:590
msgid "Question"
msgstr "Pregunta"

#: openplotterNetwork.py:613 openplotterNetwork.py:614
msgid "wifi access point: "
msgstr "punt d'accés wifi: "

#: openplotterNetwork.py:631
msgid "wifi client: "
msgstr "client wifi: "

#: openplotterNetwork.py:644
msgid "running"
msgstr "en execució"

#: openplotterNetwork.py:645
msgid "stopped"
msgstr "aturat"

#: openplotterNetwork.py:659
msgid "IP address\n"
msgstr "Adreça IP\n"

#: openplotterNetwork.py:672
msgid "Installing Wifi modules... "
msgstr "Instal·lant mòduls Wifi... "

#: openplotterNetwork.py:674
msgid "Done."
msgstr "Fet."

#: openplotterNetwork.py:712
msgid "There are conflicts between server connections"
msgstr "Hi ha conflictes entre connexions al servidor"

#: openplotterNetwork.py:713
msgid "No conflicts between server connections"
msgstr "No hi ha conflictes entre connexions al servidor"

#: ports.py:28
msgid "VNC Remote Desktop"
msgstr "Escriptori remot VNC"

#: startup.py:46
msgid "Checking WIFI Access Point password..."
msgstr "Comprovant la contrasenya del Punt d'Accés WIFI..."

#: startup.py:68
msgid "Security warning: You are using the default WIFI Access Point password.\n"
"Please change password in OpenPlotter Network."
msgstr "Alerta de seguretat: Estàs utilitzant la contrasenya per defecte del Punt d'Accés WIFI.\n"
"Si us plau canvia la contrasenya a l'aplicació OpenPlotter \"Network\"."

#: startup.py:69
msgid "changed"
msgstr "modificat"

